### PR TITLE
update license to include 2019

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
update license to include 2019 as license valid time. Shall raise an issue to fix this as it can be easily overlooked and a permanent solution needs to be done.

- **What does this PR do?**
update the license to include 2019 as covered year too
- **Any background context you want to provide?**

- **Screenshots and/or Live Demo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,21 +56,21 @@ export default class App extends Component {
   <tr>
     <td>numInputs</td>
     <td>number</td>
-    <td><strong>true</strong></td>
+    <td>true</td>
```
Comment (apollonian) [inline README.md]: I think this was intentional. Can we keep the strong tags in place. I'll merge the PR right away!

Comment (Parikshit-Hooda) [reply README.md]: HI @apollonian it makes sense to remove the strong. Please state the reason why they could have been emphasized. I shall open a new issue regarding this, fix it by keeping the strong tags in place and make a pr again.

Comment (Parikshit-Hooda) [reply README.md]: Please look for the pr

**Comment by Parikshit-Hooda** [comment]
Hi, again. The requested changes concerning the strong tags have been made.

**Pull request merged**